input/pagecontent/iti-mhd.md

```diff
@@ -28,8 +31,48 @@ transactions between them.
 The Find Document Lists [[ITI-66]](https://profiles.ihe.net/ITI/MHD/ITI-66.html) transaction defined in [MHD](https://profiles.ihe.net/ITI/MHD/index.html) SHALL not be made available in this context.
 
 ### Actor options  
-For all actors the Comprehensive Metadata Option SHALL be supported. For all actors the Metadata as defined in Annex 3 SHALL be supported.
 
+Options that can be selected for each actor in this profile,are listed in the table below. 
+
+{:class="table table-bordered"}
+| Actor                                         | Option Name         | Optionality  |
+|-----------------------------------------------|---------------------------|-------------|
+| Document Source                               | Comprehensive Metadata <br />Federated Cross Community Access   | R  <br /> see actor option notes    |
+| Document Recipient                            | Comprehensive Metadata <br />Federated Cross Community Access    | R <br /> see actor option notes |
+| Document Consumer                             | Comprehensive Metadata <br />Federated Cross Community Access   | R <br /> R  |
+| Document Responder                            | Comprehensive Metadata <br />Federated Cross Community Access  | R<br /> see actor option notes  |
+
+<figcaption ID="1">Table 1: Actor options.</figcaption>
+
+
+#### Comprehensive Metadata Option
+
+Comprehensive Metadata as defined in [1:33.2.1](https://profiles.ihe.net/ITI/MHD/1332_actor_options.html#13321-comprehensive-metadata-option). For all actors the Metadata as defined in Annex 3 SHALL be supported.
+
+#### Federated Cross Community Access Option
+
+The Federated Cross Community Access Option allows querying, retrieving documents and updating document metadata over multiple communities. 
+
+##### Querying, retrieving documents and updating document metadata over multiple communities
+
+ A community and reference community SHALL support one Document Responder with the Federated Cross Community Access and one Document Responder without the Federated Cross Community Access option. 
```

Comment:
Why 2? The Document Responder without the FXCA option shall also be exposed to clients?
Otherwise, it's an implementation detail.